x-pack/filebeat/module/kubernetes/envoy/manifest.yml

```diff
@@ -0,0 +1,11 @@
+module_version: 1.0
+
+var:
+  - name: paths
+    default:
+      - /var/lib/docker/containers/*/*-json.log
```

Comment:
This will take logs from all containers running in the system, we probably want to set here some safe default, or nothing at all so setting something is required.
Take into account that if this module is used with kubernetes or docker autodiscover this path will be easily set to the one of the specific envoy container.

Reply:
Please see file ingest-k8s.yaml and pull 10911 for addressing this. Note that the file names are using some generated 64-byte id here hence hard to specify without auto-discovery,.